src/backend/utils/init/postinit.c

```diff
@@ -536,6 +536,8 @@ BaseInit(void)
 	InitFileAccess();
 	smgrinit();
 	InitBufferPoolAccess();
+
+	InitCodeGen();
```

Comment:
Won't it be necessary to include the appropriate header? Also, we should check the return value to make sure that this went OK.